packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html

```diff
@@ -9,71 +9,79 @@
-            <oui-spinner data-size="s" data-ng-show="bridged.loading">
-            </oui-spinner>
-        </span>
-        <button
-            type="button"
-            class="oui-popover-button"
-            data-oui-popover="{{:: 'vrack__assigned-ipv6__subrange_slaac_help' | translate }}"
-            data-oui-popover-placement="right"
-            data-oui-popover-scope="$ctrl"
-            data-oui-popover-template="vrack__assigned-ipv6__subrange_slaac_help.html"
-        ></button>
-        <script
-            type="text/ng-template"
-            id="vrack__assigned-ipv6__subrange_slaac_help.html"
+    <ul class="vrack__assigned-ipv6__sub-item">
+        <li
+            class="vrack__assigned-ipv6__subrange"
+            data-ng-repeat="bridged in $ctrl.bridgedSubranges track by $index"
         >
-            <p
-                data-translate="vrack__assigned-ipv6__subrange_slaac_help"
-            ></p>
-            <a
-                class="oui-link_icon"
-                aria-hidden="true"
-                data-ng-href="{{$ctrl.slaacGuidesLink}}"
-                target="_blank"
-            >
-                <span class="oui-icon oui-icon-book ml-0" aria-hidden="true"></span>
-                <span
-                data-translate="vrack__assigned-ipv6__subrange_slaac_help_tooltip_link"
-            ></span>
-            </a>
-        </script>
-    </li>
-    <ul
-        class="vrack__assigned-ipv6__sub-item"
-        data-ng-repeat="subnet in $ctrl.subnets track by $index"
-    >
-        <!-- subnet -->
-        <li class="vrack__assigned-ipv6__subnet">
             <span
-                class="vrack__assigned-ipv6__subnet__title"
-                data-translate="vrack__assigned-ipv6_subnet_via"
-                data-translate-values="{ ipblock: $ctrl.ip.niceName, ip: subnet }"
+                class="vrack__assigned-ipv6__subrange__title"
+                data-ng-bind="::bridged.bridgedSubrange"
             ></span>
-            <oui-button
-                size="s"
-                variant="link"
-                ng-click="$ctrl.openDeleteSubnetModal(subnet)"
+            <span>
+                {{:: $ctrl.label.SLAAC }}
+                <oui-switch
+                    data-ng-hide="bridged.loading"
+                    model="bridged.model"
+                    on-change="$ctrl.toggleSubrange(bridged)"
+                >
+                </oui-switch>
+                <oui-spinner data-size="s" data-ng-show="bridged.loading">
+                </oui-spinner>
+            </span>
+            <button
+                type="button"
+                class="oui-popover-button"
+                data-oui-popover="{{:: 'vrack__assigned-ipv6__subrange_slaac_help' | translate }}"
+                data-oui-popover-placement="right"
+                data-oui-popover-scope="$ctrl"
+                data-oui-popover-template="vrack__assigned-ipv6__subrange_slaac_help.html"
+            ></button>
+            <script
```

Comment:
Better to do a component no ?

Reply:
like as other implementation of popover I implement this one but if you think is more better to create new component no problem